scripts/release.sh

```diff
@@ -110,13 +110,13 @@ fi
 echo "exporting Provider Schema JSON"
 (
   set -x
-  go run internal/tools/schema-api/main.go -export azurermProviderSchema.json
+  go run internal/tools/schema-api/main.go -export .release/azurermProviderSchema.json
```

Comment:
being a little pedantic, but if we're changing this anyway:

```suggestion
  go run internal/tools/schema-api/main.go -export .release/provider-schema.json
```

since most of the other scripts/tooling is `snake-cased`?